native/src/routes/Pois.tsx

```diff
@@ -90,7 +90,7 @@ const Pois = ({ pois: allPois, language, cityModel, route, navigation }: PoisPro
   }
 
   const deselect = () => {
-    if (multipoi && slug) {
+    if (multipoi !== undefined && slug) {
```

Comment:
💭 I understand currently why we're treating `multipoi` and `slug` differently here but I'm not sure I will understand that in the future. But I guess the linter will yell at me if I use the check incorrectly.